cpp-ch/local-engine/Parser/SerializedPlanParser.cpp

```diff
@@ -1656,6 +1629,86 @@ ActionsDAGPtr SerializedPlanParser::parseArrayJoin(
     return actions_dag;
 }
 
+ActionsDAGPtr SerializedPlanParser::parseJsonTuple(
+    const Block & input,
+    const substrait::Expression & rel,
+    std::vector<String> & result_names,
+    std::vector<String> & required_columns,
```

Comment:
the `required_columns` is not used in this function, remove it. since all the `required_columns` will be removed in #1901

Reply:
done